src/mavedb/lib/script_environment.py

```diff
@@ -0,0 +1,33 @@
+"""
+Environment setup for scripts.
+"""
+
+import logging
+
+from sqlalchemy.orm import configure_mappers, Session
+
+from mavedb import deps
+from mavedb.models import *
+
+def init_script_environment() -> Session:
+    """
+    Set up the environment for a script that may be run from the command line and does not necessarily depend on the
+    FastAPI framework.
+
+    Features:
+    - Configures logging for the script.
+    - Loads the SQLAlchemy data model.
+    - Returns an SQLAlchemy database session.
+    """
+
+    logging.basicConfig()
+
+    # Un-comment this line to log all database queries:
+    logging.getLogger("__main__").setLevel(logging.INFO)
```

Comment:
is the `# Un-comment this line` comment referring to the line it immediately precedes or the next one? In which logging config is this code intended to be submitted?